src/main/ruby/truffleruby/core/file.rb

```diff
@@ -37,9 +37,12 @@
 class File < IO
   include Enumerable
 
-  class FileError < Exception; end # rubocop:disable Lint/InheritException
+  class FileError < RuntimeError; end
+
   class NoFileError < FileError; end
+
   class UnableToStat < FileError; end
+
   class PermissionError < FileError; end
```

Comment:
It seems these exception classes aren't used anywhere. Should we just remove them?

cc @eregon